modules/dataquery/jsx/react.app.js

```diff
@@ -466,6 +466,26 @@ var DataQueryApp = React.createClass({
                 loading: false
             }
         });
+        for (var i = 0; i < fieldsList.length; i++) {
+            var that = this;
+            $.ajax({
+                url: loris.BaseURL + "/AjaxHelper.php?Module=dataquery&script=datadictionary.php",
```

Comment:
This URL should be `loris.BaseURL + "/dataquery/ajax/datadictionary.php"`, not directly calling `AjaxHelper`